qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -189,13 +190,10 @@ def __init__(self, *, win_id, private, parent=None):
         self._hbox.addWidget(self.url)
```

Comment:
Can you add `{url}` or similar to the widget list, some people might want to put their url on the right, for example.

Reply:
Sure